samples/storageclass/powermax.yaml

```diff
@@ -41,6 +41,23 @@ parameters:
   # Optional: true, Default value: None
   # Examples: APP, app, sanity, tests
   ApplicationPrefix: <application prefix>
+  # HostLimitName uniquely identifies given set of limits on a storage class
+  # This is used in naming storage group, max of 3 letter
+  # Optional: true
+  # Example: "HL1", "HL2"
+  HostLimitName: "HL1"
```

Comment:
can we comment these fields and add the description like add these only if want host limits on SG/SC's